### PR TITLE
greenfield and brownfield installation doc update

### PR DESCRIPTION
**Pull request by 3quanfeng**
## Checklist
- [x] The title of the PR is clear and informative
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] Issues addressed by the PR are mentioned in the description followed by `Fixes`.

## Description

- update main readme page to include release notes

- update doc greenfield and brownfield installation

- add troubleshooting guide when AGIC fails to work with AAD Pod Identity due to its break change

- reorg the troubleshooting docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ AGIC is configured via the Kubernetes [Ingress resource](http://kubernetes.io/do
   - Support for public, private, and hybrid web sites
   - Integrated web application firewall
 
+## Change notes
```
Comment (akshaysngupta) [inline README.md:21]: Changelog ?

File: docs/setup/install-new.md [hunk 2]
```diff
@@ -162,6 +162,7 @@ az aks get-credentials --resource-group $resourceGroupName --name $aksClusterNam
   ```bash
   kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.5.5/deploy/infra/deployment.yaml
   ```
+  ***Note:*** AAD Pod Identity introduces a [breaking change](https://github.com/Azure/aad-pod-identity/tree/v1.6.0#v160-breaking-change) after v1.5.5 regarding CRD fields become case sensitive, for any AAD Pod Identity version >= 1.6.0 or you plan to apply from such as https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/deployment-rbac.yaml, AGIC version at least [v1.2.0-rc2](https://github.com/Azure/application-gateway-kubernetes-ingress/blob/master/CHANGELOG/CHANGELOG-1.2.md#v120-rc2) will be required.
```
Comment (akshaysngupta) [inline docs/setup/install-new.md:165]: `introduces` -> `introduced`

File: docs/setup/install-new.md [hunk 3]
```diff
@@ -162,6 +162,7 @@ az aks get-credentials --resource-group $resourceGroupName --name $aksClusterNam
   ```bash
   kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.5.5/deploy/infra/deployment.yaml
```
Comment (akshaysngupta) [inline docs/setup/install-new.md:163]: we need to change 1.5.5 to 1.6.0. 1.5.5 will not work anymore.

Comment (3quanfeng) [reply docs/setup/install-new.md:163]: Updated

Comment (akshaysngupta) [inline docs/setup/install-new.md:163]: we should link to AAD 1.6

Comment (akshaysngupta) [reply docs/setup/install-new.md:163]: https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.6.0/deploy/infra/deployment.yaml

Comment (akshaysngupta) [reply docs/setup/install-new.md:163]: I think we might have to change in other files also

Comment (akshaysngupta) [reply docs/setup/install-new.md:163]: and also, we should update the installation version to 1.2.0
```
helm install ingress-azure \
      -f helm-config.yaml \
      application-gateway-kubernetes-ingress/ingress-azure
      --version 1.2.0
```

File: docs/troubleshootings/troubleshooting-agic-addon-identity-not-found.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+## Troubleshooting: AGIC addon, identity not found
+
+### Illustration
+AGIC fails to start due to "Identity not found" even though the required permissions are provided.
```
Comment (akshaysngupta) [inline docs/troubleshootings/troubleshooting-agic-addon-identity-not-found.md:4]: Add space after this line. Next title is getting rendered in the same line.

Comment (akshaysngupta) [inline docs/troubleshootings/troubleshooting-agic-addon-identity-not-found.md:4]: How about changing this to ?
`AGIC fails to start due to "Identity not found" even though the User Managed Identity to the AKS nodes is correctly assigned and it has the right permissions.`

File: docs/setup/install-new.md (hunk 3 above)
Comment (3quanfeng) [reply docs/setup/install-new.md:163]: updated

**Pull request merged**